salman_hossain/class_four/max.rb

```diff
@@ -0,0 +1,30 @@
+My_nums = [-10, -9, -8, -7, -6, -5, 0, 1, 2, 3, 4, 5, 100, 200, 300, 400, 500, 600, 700, 800, 900, 10000, 200000]
+
+# puts My_nums.min 
+
+def my_max(nums)
+   current_max = nums.first
+   nums.each do |x|
+       if x >= current_max
```

Comment:
you could save yourself an operation here and just put:

`if x > current_max`

so if the number is the same, the loop will skip instead of reassigning the current_max variable again :+1: